control-plane-operator/controllers/hostedcontrolplane/kas/config.go

```diff
@@ -194,11 +194,8 @@ func generateConfig(p KubeAPIServerConfigParams, version semver.Version) *kcpv1.
 	args.Set("enable-aggregator-routing", "true")
 	args.Set("enable-logs-handler", "false")
 	args.Set("endpoint-reconciler-type", "lease")
-	args.Set("etcd-cafile", cpath(kasVolumeEtcdCA().Name, certs.CASignerCertMapKey))
-	args.Set("etcd-certfile", cpath(kasVolumeEtcdClientCert().Name, pki.EtcdClientCrtKey))
-	args.Set("etcd-keyfile", cpath(kasVolumeEtcdClientCert().Name, pki.EtcdClientKeyKey))
 	args.Set("etcd-prefix", "kubernetes.io")
-	args.Set("etcd-servers", p.EtcdURL)
+	args.Set("etcd-servers", "http://127.0.0.1:23790")
```

Comment:
KAS <-> etcd-proxy traffic is in-the-clear and unauthenticated over localhost?

Reply:
Yes it is. It is internal to the loopback interface of the pod, so I didn't see the need for the added overhead of encrypted traffic.

Reply:
After chatting with Derek, I will update this endpoint to use encryption given that data can still be sniffed with a traffic analyzer.

Reply:
Updated now to use encrypted communication between KAS container and etcd-proxy